client/src/__snapshots__/ElementsProviderCheckout.test.js.snap

```diff
@@ -67,3 +67,41 @@ exports[`<ElementsProviderCheckout/> renders Stripe's Elements provider with cli
   </div>
 </DocumentFragment>
 `;
+
+exports[`<ElementsProviderCheckout/> renders payment form when clientSecret is present 1`] = `
```

Comment:
look into test that was probably duplicated, we shouldn't have 2 snapshots for this test case

Reply:
done